src/xmipp/libraries/parallel/mpi_image_sort.cpp

```diff
@@ -151,7 +150,7 @@ class ProgSortImages: public XmippProgram
             }
             if (proceed)
             {
-                toClassify.push_back(row);
+                toClassify.push_back(dynamic_cast<MDRowVec&>(row));
```

Comment:
`emplace_back({row})`?

Reply:
`dynamic_cast` needs to be there, bacause iterating over `MetaData*` always returns reference to abstract `MDRow`.

Reply:
(This isn't the best solution but I don't know ho to do it better)